create_table.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE users (
```

Comment:
do you need to initialise the db before creating it, in case there are previous tables named the same ?